packages/desktop-client/src/components/modals/PasswordEnableModal.tsx

```
@@ -0,0 +1,133 @@
+import { useState } from 'react';
+
+import { send } from 'loot-core/platform/client/fetch';
+import * as asyncStorage from 'loot-core/src/platform/server/asyncStorage';
+
+import { useActions } from '../../hooks/useActions';
+import { theme, styles } from '../../style';
+import { Error } from '../alerts';
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)